modules/ROOT/pages/bundling-plugins.adoc

```diff
@@ -27,15 +28,17 @@ Example syntax for including the example "plugin" in a bundle:
 a|
 [source, js]
 ----
-import pluginCss from 'tinymce/plugins/example/content.css';
+import pluginCss from 'tinymce/plugins/example/content.js';
+//import pluginCss from 'tinymce/plugins/example/content_css.js';
 import 'tinymce/plugins/example';
 ----
 
 |`.zip`
 a|
 [source, js]
 ----
-import pluginCss from '../tinymce/plugins/example/content.css';
+import pluginCss from '../tinymce/plugins/example/content.js';
+//import pluginCss from '../tinymce/plugins/example/content_css.js';
```

Comment:
These files do not export anything they just run some code so we need to import the whole file NOT import **_from_** the file
```suggestion
import '../tinymce/plugins/example/content.js';
//import '../tinymce/plugins/example/content_css.js';
```